weather.css

```diff
@@ -0,0 +1,54 @@
+
+#font
+ {
+    color: white;
+    font-size: 100px;
+    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  }
+  table{
```

Comment:
HTML selectors like "table,body,h1,h2 ...." are usually placed at the top of the css file because of its specificity and organization clarity.

Reply:
Understood changed it.